packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -226,6 +232,59 @@ contract VerifyOPCM is Script {
         return refs;
     }
 
+    /// @notice Verifies that all four OPCM contracts have the same contractsContainer address.
+    /// @param _opcm The live OPCM contract.
+    function _verifyContractsContainerConsistency(IOPContractsManager _opcm) internal view {
+        // Get the contractsContainer addresses from all four contracts.
+        address gameTypeAdderContainer = _getContractsContainerAddress(address(_opcm.opcmGameTypeAdder()));
+        address deployerContainer = _getContractsContainerAddress(address(_opcm.opcmDeployer()));
+        address upgraderContainer = _getContractsContainerAddress(address(_opcm.opcmUpgrader()));
+        address interopMigratorContainer = _getContractsContainerAddress(address(_opcm.opcmInteropMigrator()));
```

Comment:
One thing about this approach, is that unlike `_getOpcmPropertyRefs()`, it won't automatically detect new `opcmX()` components which might be added in the future, and I think that's a nice property this script otherwise has.

I think we can get a smaller diff by make this a check that just iterates over [`propRefs`](https://github.com/ethereum-optimism/optimism/blob/190d8771a49c6b9c122539244e556437ed779eb7/packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol#L191) to ensure they all have the same address.